src/DUWOagent/DUWO.mod2g

```diff
@@ -13,18 +13,18 @@ module DUWO {
 	
 	% Demolish when we have this as a goal
 	if goal(goalDemolish) 
-		then demolishMod.
+			then demolishMod.
 	
 	% Build student housing when have it as goal.
 	% Use the normal module when we have enough budget
-	if not(bel(lowBudget)), not(bel(noBudget))
-		then normalMod.
+	if goal(goalBuildStudentHousing)
```

Comment:
goalReachBudgetTarget is indeed a goal when nobudget is true, but buildstudenthousing can be a goal regardless of budget